<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="39954" started-at="2016-09-07T08:16:17Z" finished-at="2016-09-07T08:16:57Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="39954" started-at="2016-09-07T08:16:17Z" finished-at="2016-09-07T08:16:57Z">
      <class name="com.young.appiumdemo.AppiumScreenShotOnFailureTest">
        <test-method status="PASS" signature="startTest()[pri:0, instance:com.young.appiumdemo.AppiumScreenShotOnFailureTest@694bc02d]" name="startTest" is-config="true" duration-ms="31753" started-at="2016-09-07T16:16:17Z" finished-at="2016-09-07T16:16:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startTest -->
        <test-method status="FAIL" signature="screenshotOnFailure()[pri:0, instance:com.young.appiumdemo.AppiumScreenShotOnFailureTest@694bc02d]" name="screenshotOnFailure" duration-ms="1940" started-at="2016-09-07T16:16:49Z" finished-at="2016-09-07T16:16:51Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Shw Alert] but found [Show Alert]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Shw Alert] but found [Show Alert]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at com.young.appiumdemo.AppiumScreenShotOnFailureTest.screenshotOnFailure(AppiumScreenShotOnFailureTest.java:50)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- screenshotOnFailure -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:com.young.appiumdemo.AppiumScreenShotOnFailureTest@694bc02d]" name="afterClass" is-config="true" duration-ms="4456" started-at="2016-09-07T16:16:52Z" finished-at="2016-09-07T16:16:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- com.young.appiumdemo.AppiumScreenShotOnFailureTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
